lib/graphql/schema/argument.rb

```diff
@@ -61,6 +61,8 @@ def initialize(arg_name = nil, type_expr = nil, desc = nil, required:, type: nil
         @from_resolver = from_resolver
         @method_access = method_access
 
+        check_conflicting_params
+
```

Comment:
From a maintainability standpoint, it'd be easier for me if the code was inline, here in `#initialize` instead of separated by a jump, down at the bottom of the file. 

For a reader who wants to debug this method, they'll have to break context and go read the bottom of the file if they want to know what's going on here. 

The name, `check_conflicting_params`, doesn't make it clear which params are being checked or what kind of conflicts its looking for. Since the method accepts no arguments, the reader is invited to assume that _any_ internal state of the instance is used as input by `check_conflicting_params`. 

Do you mind removing the method and putting the validation logic here in `#initialize`?

Reply:
I'll do the changes!!